packages/lib/modules/pool/PoolDetail/PoolUserEvents/PoolUserEvents.tsx

```diff
@@ -189,11 +187,12 @@ export default function PoolUserEvents({
     }
   }, [])
 
-  useEffect(() => {
+  const poolEvents: PoolEventItem[] = useMemo(() => {
```

Comment:
why do we need the `useMemo` here? shouldn't this be similar to `const poolEvents = !isLoading ? userPoolEvents : []` or even better always return a list from `useUserPoolEvents`?